docs/src/gallery/geometries.md

```diff
@@ -13,13 +13,42 @@ p1 = plot(dataset("ggplot2", "mpg"),
      Guide.annotation(compose(context(), text(6,4, "y=x", hleft, vtop), fill("red"))))
 
 x = [20*rand(20); exp(-3)]
-D = DataFrame(x=x, y= exp.(-0.5*asinh.(x).+5) .+ 2*randn(length(x))) 
+D = DataFrame(x=x, y= exp.(-0.5*asinh.(x).+5) .+ 2*randn(length(x)))
 abline = Geom.abline(color="red", style=:dash)
 p2 = plot(D, x=:x, y=:y,  Geom.point,  Scale.x_asinh, Scale.y_log,
      intercept=[148], slope=[-0.5], abline)
 hstack(p1, p2)
 ```
 
+## [`Geom.band`](@ref), [`Geom.hband`](@ref), [`Geom.vband`](@ref)
+
+
+```@example
+using Colors, Dates, Gadfly, RDatasets
+
+Dp = dataset("ggplot2","presidential")[3:end,:]
+De = dataset("ggplot2","economics")
+De.Unemploy /= 10^3
+
+plot(De, x=:Date, y=:Unemploy, Geom.line,
+    layer(Dp, xmin=:Start, xmax=:End, Geom.vband, color=:Party),
+    Scale.color_discrete_manual("deepskyblue", "lightcoral"),
+    Coord.cartesian(xmin=Date("1965-01-01"), ymax=12),
+    Guide.xlabel("Time"), Guide.ylabel("Unemployment (x10³)"), Guide.colorkey(title=""),
+    Theme(default_color="black", key_position=:top))
+```
+
+```@example
+using Colors, Gadfly
+
+p1 = plot(x=1:9, y=1:9, Geom.blank,
+        layer(xmin=[1.0, 5.0, 7.0], xmax=[2.0, 6.5, 8.0], Geom.vband, Theme(default_color=RGBA(0.5,0.5,0.5,0.5))));
+
+p2 = plot(x=1:9, y=1:9, Geom.blank,
+          layer(ymin=[2.5], ymax=[7.5], Geom.hband, Theme(default_color=RGBA(0.5,0.5,0.5,0.5))));
+
+hstack(p1, p2)
+```
```

Comment:
this code is a duplicate of what's in the tests.  i'd suggest removing it from the gallery.

Reply:
I have removed this duplication.